sklearn/multioutput.py

```diff
@@ -636,22 +637,68 @@ def _check(self):
 
 class _BaseChain(BaseEstimator, metaclass=ABCMeta):
     _parameter_constraints: dict = {
-        "base_estimator": [HasMethods(["fit", "predict"])],
+        "base_estimator": [
+            HasMethods(["fit", "predict"]),
+            StrOptions({"deprecated"}),
+        ],
+        "estimator": [
+            HasMethods(["fit", "predict"]),
+            None,
```

Comment:
We can use the `Hidden` option here. You can import it from `sklearn.utils._param_validation` module.

```suggestion
            Hidden(None),
```